docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md

```diff
@@ -52,21 +52,21 @@ To create a discount voucher:
       {% endinfo_block %}
 
     2. Select the **Discount application type** and define the products to which the voucher should be applied. See reference information of the [Discount calculation](#discount-calculation-tab) tab for more details.
- 3. Click **Next**, or select the *Conditions* tab to proceed.
- 4. On the *Create new discount* page, in the *Conditions* tab, do the following:
-    1. Select the **Apply when** conditions or click **Plain query** and enter the  query manually. See reference information of the [Conditions](#conditions) tab for more details.
-    2. Enter the value for the *THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).* field.
-3. Click **Save** to create the new voucher.
+5. Click **Next**, or select the **Conditions** tab to proceed.
+6. On the **Conditions** tab, do the following:
+    1. Select the **APPLY WHEN** conditions or click **Plain query** and enter the  query manually. See reference information of the [Conditions](#conditions) tab for more details.
+    2. Enter the value for the **THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).** field.
+7. To create the new voucher, click **Save**.
 
-When you click **Save**, an additional tab named *Voucher Codes* appears. Here, you can generate, view, and export voucher codes (if they were already created).
+When you click **Save**, an additional tab named **Voucher Codes** appears. Here, you can generate, view, and export voucher codes (if they were already created).
 The list is empty until codes are generated.
 
-In the *Voucher code* tab, do the following:
-1. Enter the *QUANTITY* for voucher codes you want to generate.
-2. _Optional_: Enter a *CUSTOM CODE*.
-3. Set the *ADD RANDOM GENERATED CODE LENGTH* by selecting the value from a drop-down list.
-4. Set *MAX NUMBER OF USES*.
-5. Click **Generate** to complete the process.
+On the **Voucher code** tab, do the following:
```

Comment:
```suggestion
In the **Voucher code** tab, do the following:
```

Reply:
According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.